README.md

```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
+The file storing the context details (i.e. `bunq.conf`) is a key to your account. Anyone having access to it is able to perform any Public API actions with your account. Therefore, we recommend choosing a truly safe place to store it.
+
+If you rather save the context in a database, you can use the `fromJson()` and `toJson()` methods. 
 
 ### Making API calls
 There is a class for each endpoint. Each class has functions for each supported action. These actions can be
 `create`, `get`, `update`, `delete` and `listing`.
 
-Sometimes API calls have dependencies, for instance `MonetaryAccount`. Making changes to a monetary account 
-always also needs a reference to a `User`. These dependencies are required as arguments when performing API calls.
+Before you can start making calls, you must ensure that you have create an ApiContext and loaded in into BunqContext as shown in the examples above. 
+
+The SDK will take care of your user Id, as this id wil never change per ApiContext. The SDK also uses your first active monetary account as primary monetary account. This is alsmot always the same as your billing account. This means, if you do not explicitly pass a monetary account id, the SDK will use the billing account id.  
+
 Take a look at [doc.bunq.com](https://doc.bunq.com) for the full documentation.
 
 #### Creating objects
-Creating objects through the API requires an `ApiContext`, a `requestMap` and identifiers of all dependencies (such as
-User ID required for accessing a Monetary Account). Optionally, custom headers can be passed to requests.
-
 ```php
-$paymentMap = [
-    Payment::FIELD_AMOUNT => new Amount('10.00', 'EUR'),
-    Payment::FIELD_COUNTERPARTY_ALIAS => new Pointer('EMAIL', 'api-guru@bunq.io'),
-    Payment::FIELD_DESCRIPTION => 'This is a generated payment',
-];
-
-Payment::create($bunqApiContext, $paymentMap, $userId, $monetaryAccountId);
+BunqContext::loadApiContext($apiContext); // if it has not been loaded yet. 
+
+Payment::create(
+            new Amount($amount, self::CURRENCY_TYPE_EUR),
+            new Pointer(self::POINTER_TYPE_EMAIL, $recipient),
+            $description,
+            $monetaryAccount->getId()
+        )
 ```
 
 ##### Example
-See [`example/payment_example.php`](./example/payment_example.php)
+See [`tinker/BunqLib`](https://github.com/bunq/tinker_php/blob/05a38a2660e6f6db1f7efc9b915f0131c172c230/src/BunqLib.php#L240-L245)
 
 #### Reading objects
-Reading objects through the API requires an `ApiContext`, identifiers of all dependencies (such as User ID required for
-accessing a Monetary Account), and the identifier of the object to read (ID or UUID) Optionally, custom headers can be
-passed to requests.
+To use the read method you must pass the identifier of the object to read (ID or UUID) expect for the endpoints User, UserPerson, UserCompany and monetary account. As the SDK will use default ids if not passed. For all other endpoints you must pass the identifier. 
```

Comment:
`expect` ➡️  `except`

`monetary account` ➡️  `Monetary Account`

`As the SDK will use default ids if not passed` ➡️  `The SDK will use the default IDs when none are passed.`